#: component.py:95
msgid "NextGIS geoinformation system"
msgstr "Geoinformační systém NextGIS"

#: component.py:276
msgid "Linux kernel"
msgstr "Linuxové jádro"

#: component.py:279
msgid "OS distribution"
msgstr "Distribuce OS"

#: component.py:291
msgid "CPU"
msgstr "CPU"

#: component.py:294
msgid "RAM"
msgstr "RAM"

#: component.py:323
msgid "Last backup"
msgstr ""

#: component.py:327
msgid "Last maintenance"
msgstr ""

#: exception.py:116
msgid "Validation error"
msgstr "Chyba ověření"

#: exception.py:121
msgid "Forbidden"
msgstr "Zakázán"

#: exception.py:126
msgid "Insufficient permissions"
msgstr "Nedostatečná oprávnění"

#: exception.py:131
msgid "Operational error"
msgstr "Provozní chyba "

#: exception.py:136
msgid "External service error"
msgstr "Chyba externí služby"

#: exception.py:145
msgid "Not configured"
msgstr ""

#: fontconfig.py:112
msgid "Font not found: {}!"
msgstr ""

#: storage.py:290
msgid "Storage full"
msgstr "Skladování plné"

#: storage.py:291
msgid ""
"This operation can't be performed because the storage limit (%s) has been "
"already exceeded by %s."
msgstr ""
"Tuto operaci nelze provést, protože limit úložiště (%s) byl již překročen o "
"%s."

#: storage.py:301
msgid "Not enough storage"
msgstr "Nedostatek úložného prostoru"

#: storage.py:302
msgid ""
"This operation requires %s of storage and can't be performed because only %s "
"available."
msgstr ""
"Tato operace vyžaduje %s úložiště a nelze ji provést, protože je k dispozici "
"pouze %s."

#: storage.py:312
msgid "Estimation is already running"
msgstr "Odhad již běží"

#: test/test_exception.py:76
msgid "The answer is %d"
msgstr "Odpověď je %d"
